docs/usage/import-files.md

```diff
@@ -62,6 +62,10 @@ Expanding the scope of file imports, users can seamlessly add files in the `Cont
 
 In order to initiate file imports, users must possess the requisite [capability](../administration/users.md): "Upload knowledge files." This capability ensures that only authorized users can contribute and manage knowledge files within the OpenCTI platform, maintaining a controlled and secure environment for data uploads.
 
+**Max confidence level**
+
+As explained in this [page](https://docs.opencti.io/latest/usage/reliability-confidence/), it is important to set a max confidence level for your users in order to be able to create, update or delete knowledge.
```

Comment:
I don't think it's relevant to put this information on this page

Reply:
It's fine for me, I'll remove this part.